src/services/smartWallet.js

```diff
@@ -537,6 +539,13 @@ class SmartWallet {
     return formatEstimated(estimated);
   }
 
+  estimateAccountDeployment() {
+    return this.getSdk().estimateAccountDeployment().catch((err) => {
```

Comment:
If you catch the error here, then you won't be able to catch it again on the  top level

Reply:
Was moving things around and left, fixed now, thanks 👍